compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -201,29 +406,58 @@ struct QuantizeDequantizeWeights final : public luci::CircleNodeMutableVisitor<b
       {
         auto circle_const = loco::must_cast<luci::CircleConst *>(circle_node);
 
+        // Find min/max per channel-wise
+        if (granularity == QuantizationGranularity::ChannelWise)
+        {
+          std::vector<float> min;
+          std::vector<float> max;
+
+          cal_minmax_per_channel(circle_const, min, max);
+
+          std::vector<float> nudged_min(min.size());
+          std::vector<float> nudged_max(min.size());
+          std::vector<float> scaling_factor(min.size());
+          std::vector<int64_t> zp(min.size());
+
+          wquant_per_channel(circle_const, min, max, scaling_factor, zp, nudged_min, nudged_max);
+          wdequant_per_channel(circle_const, scaling_factor, nudged_min);
+
+          auto quantparam = std::make_unique<CircleQuantParam>();
+          for (size_t i = 0; i < min.size(); ++i)
+          {
+            quantparam->min.push_back(nudged_min[i]);
+            quantparam->max.push_back(nudged_max[i]);
+            quantparam->scale.push_back(scaling_factor[i]);
+            quantparam->zerop.push_back(zp[i]);
+          }
```

Comment:
How about this change?
```suggestion
          quantparam->min = nudged_min;
          quantparam->max = nudged_max;
          quantparam->scale = scaling_factor;
          quantparam->zerop = zp;
```